clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/prefer-at-over-subscript-operator.rst

```diff
@@ -0,0 +1,31 @@
+.. title:: clang-tidy - cppcoreguidelines-prefer-at-over-subscript-operator
+
+cppcoreguidelines-prefer-at-over-subscript-operator
+=====================================
+
+This check flags all uses of ``operator[]`` where an equivalent (same parameter and return types) ``at()`` method exists and suggest using that instead.
```

Comment:
Please synchronize with statement in Release Notes.

Reply:
Which of the two statements do you prefer: this one or the one in the release notes?

> Please synchronize with statement in Release Notes.

Which of the two statements do you prefer: this one or the one in the release notes?

Reply:
One in Release Notes. You could add more explanations in other sentences if needed.